pkg/controller/deployment/deployment_controller.go

```diff
@@ -191,7 +197,7 @@ func (dc *DeploymentController) deleteDeployment(obj interface{}) {
 			return
 		}
 	}
-	klog.V(4).Infof("Deleting deployment %s", d.Name)
+	klog.V(4).InfoS("Deleting deployment", klogKeyDeployment, klog.KObj(d))
```

Comment:
similar, we can remove the duplication of `deployment`

Reply:
same as above (line 175)